### ISSUE TITLE
Autobahn Police Simulator (348510)

### ISSUE THREAD

**Issue by Newbytee** [ValveSoftware/Proton#3716]
# Whitelist Request
- Name of the game to be whitelisted: Autobahn Police Simulator
- Steam AppID of the game: 348510

## System Information
- GPU: GTX 970
- Driver/LLVM version: nvidia 440.64
- Distro version: Fedora 31
- Link to full system information report as [Gist](https://gist.github.com/):
- Proton version: 5.0-5b

## I confirm:
- [X] that pressing the `Play` button in the Steam client is sufficient.
- [ ] that [runtime config options](https://github.com/ValveSoftware/Proton#runtime-config-options) are necessary to run the game.
- [ ] that no workarounds other than PROTON_* env variables are necessary.

## Issues
- [X] I haven't experienced any issues.
- [ ] There are no issues left open for this game.
- [ ] Although I consider the gaming experience equal to Windows there are remaining issues:

**Comment by shoober420** [comment]
# Whitelist Request
- Name of the game with compatibility issues: Autobahn Police Simulator
- Steam AppID of the game: https://steamdb.info/app/348510/

## System Information
- GPU: GTX 760
- Driver/LLVM version: NVIDIA 440.64
- Kernel version: 5.5.13-zen
- https://github.com/shoober420/linux-scripts/blob/main/home/shoober420/sysinfonvidia
- Proton version: 5.0-5b

LOG:
[steam-348510dx.log](https://github.com/ValveSoftware/Proton/files/4419120/steam-348510dx.log)

DX9 LOG:
[steam-348510.log](https://github.com/ValveSoftware/Proton/files/4419121/steam-348510.log)

## Symptoms
The game runs, but fps is around 10-15. I've read that this game suffers from poor performance optimizations on Windows as well, so I'm not sure if this is to be expected.

**Comment by Newbytee** [comment]
Yeah, I had pretty garbage performance (hanging around in the upper 20s and lower 30s) too if I turned it up to the highest graphics settings (still at 1080p though). On the default/recommended settings I hit a locked 60 FPS at 1080p though. I presumed this was due to poor optimisation of the game itself rather than the fault of Wine/DXVK due to a good number of Steam reviews mentioning poor optimisation/performance, but it would be great to have someone verify that this is the case.

Select reviews mentioning poor performance: 
 - https://steamcommunity.com/id/therealimnotmental/recommended/348510/
 - https://steamcommunity.com/profiles/76561198338930384/recommended/348510/
 - https://steamcommunity.com/id/yga24/recommended/348510/

And, while I can't check the first guy, the last 2 definitely seem to be playing on Windows according to SteamDB. See the "Hours per platform" table: 
 - https://steamdb.info/calculator/76561198338930384/
 - https://steamdb.info/calculator/76561198141101093/

**Comment by shoober420** [comment]
@Newbytee 
When using default video settings, my performance did improve drastically. It was around 30 FPS. My computer is assumably a lot older than yours (I have a Core 2 Quad Q9560), so this is all normal then. You're right, its just a badly optimized game lol.
